scripts/s_aureus/collate_results.py

```diff
@@ -0,0 +1,65 @@
+import os
+import json
+import csv
+
+def collate_results():
+    # Merge the resultfiles.json files
+    output_resultfiles_json_dict = dict()
+    for f in os.listdir():
+        if 'resultfiles.json' not in f or str(f) == 'resultfiles.json':
+            print('Skipping: ' + str(f))
+            continue
+        print('Loading: ' + str(f))
+        with open(f) as channel_json_file:
+            channel_dict = json.load(channel_json_file)
+            output_resultfiles_json_dict.update(channel_dict)
+
+    # Find duplicate files (Framework will error if 2 runs point to the same file)
+    all_files = set()
+    dup_files = set()
+    for sam_key, sam_body in output_resultfiles_json_dict.items():
+        for i in sam_body:
+            if i['file'] in all_files:
+                dup_files.add(i['file'])
+            all_files.add(i['file'])
+    if dup_files:
+        print('WARNING found duplicate files will remove them: ' + str(dup_files))
```

Comment:
Can you explain how duplicate files can be possible?

Reply:
The following files are duplicates

      5       "file": "s3://psga-dev-bucket-results/dev/f25f2e86-96eb-40b1-98c1-b855c817e7ca/bactopia_one/local-single-sample/bactopia/nf-reports/bactopia-trace.txt",
      5       "file": "s3://psga-dev-bucket-results/dev/f25f2e86-96eb-40b1-98c1-b855c817e7ca/bactopia_one/local-single-sample/logs/custom_dumpsoftwareversions/versions.yml",
      5       "file": "s3://psga-dev-bucket-results/dev/f25f2e86-96eb-40b1-98c1-b855c817e7ca/bactopia_one/local-single-sample/software_versions.yml",
      5       "file": "s3://psga-dev-bucket-results/dev/f25f2e86-96eb-40b1-98c1-b855c817e7ca/bactopia_one/quast_assembly.zip",

Quick fix is to just not return these files to the user

Reply:
They contain data that is constant for all samples in a run